### PR TITLE
Remove extra slash from builder that is not handled on windows

### PR DESCRIPTION
**Pull request by jonahwilliams**
~~Still need to confirm this is the cause on windows machines.~~ Build succeeds but the builder does not output the file correctly, investigating that as an issue with the builder in question.

### CODE REVIEW

**Comment by jonahwilliams** [comment]
Fixes https://github.com/flutter/flutter/issues/28026

File: packages/flutter_tools/lib/src/codegen.dart [hunk 1]
```diff
@@ -291,3 +301,10 @@ class CodeGeneratingResidentCompiler implements ResidentCompiler {
     return _residentCompiler.shutdown();
   }
 }
+
```
Comment (goderbauer) [inline packages/flutter_tools/lib/src/codegen.dart]: extra blank line?

File: packages/flutter_tools/lib/src/codegen.dart [hunk 2]
```diff
@@ -291,3 +301,10 @@ class CodeGeneratingResidentCompiler implements ResidentCompiler {
...
+
+enum CodegenStatus {
```
Comment (goderbauer) [inline packages/flutter_tools/lib/src/codegen.dart]: Missing doc comments?

**Review by goderbauer — APPROVED** [review]
LGTM

**Pull request merged**